tests/units/cli/test_commands.py

```diff
@@ -662,3 +678,189 @@ def test_run_command(self):
             )
         assert os.path.isfile('test.log.failed')
         os.remove('test.log.failed')
+
+
+class TestConfigValidation:
```

Comment:
I see there is a lot of repetition of creating valid and invalid files, maybe it's better to create one valid, one invalid and 1 empty file and reuse them throughout the test suite.

Reply:
This helps each test be isolated and using a new file and also in some scenarios an invalid file will be fixed and become a valid one.